doc/CMakeLists.txt

```diff
@@ -1,76 +1,99 @@
-#
 # Copyright (c) 2019 Nordic Semiconductor ASA
-#
 # SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+
+# Builds combined documentation for nRF, Zephyr, MCUboot, and nrfxlib.
```

Comment:
Probably safer to say "for all doc sets", since this will change (and noone will remember to update this line ;-) ).

Reply:
Changed. Listed all of them except nrfxlib and added "etc.", to still show examples.